<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RootWidget</class>
 <widget class="QWidget" name="RootWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RootWidget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <widget class="QWidget" name="pageMain">
    <widget class="QPushButton" name="buttonSettings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>85</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonMore">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>85</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>More</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSettingsData">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Settings data:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>More data:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMoreData">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageSettings">
    <widget class="QPushButton" name="buttonBack">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>260</y>
       <width>85</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Item 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Item 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Item 3</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
